DeploymentExperiences.md

```diff
@@ -0,0 +1,7 @@
+## Deployment Experience
+
+Was quite sick for half the day on Tuesday and so I missed the djorg stuff and couldn't concentrate well in the morning part. When I was feeling better on Wednesday, I was keeping up with that day's lecture. But, as of yesterday I went back to Djorg and am making progress with completing that, so it looks i'll be back on track by this evening. So I can't answer this completely at the moment, but probably by evening or tomorrow. I'm making good progress on Djorg. As for deployment instructions it seems somewhat clear to me of what needs to be done, but I'll have a better idea when I'm at that point.
+
```

Comment:
Keep it up, I hope to see a deployed site soon. I know how hard it is to catch up after being sick.